### DISCUSSION TITLE
Do user commands prefixed with Mason* get cleared by the plugin?

### DISCUSSION THREAD

**Discussion by siduck** [mason-org/mason.nvim#1795] [Q&A]
Hi, i made a command `MasonInstallAll` and it gets deleted after i open mason window!

https://github.com/user-attachments/assets/6874afaf-aafa-4ff1-9565-67287b1e7cf9

**Comment by siduck** [comment]
@williamboman  any idea?

**Comment by willnode** [comment]
I'm surprised too... My new install with nvchad made MasonInstallAll disappear after opening mason window.

**Comment by wfaler** [comment]
Any fixes/workarounds anyone found for this?

**Reply by siduck** [reply]
@wfaler just map `require('nvchad.mason').install_all()` function to some command

**Comment by siduck** [comment]
@wfaler it was a nvchad bug, do lazy sync @willnode
